fenix/app/src/main/java/org/mozilla/fenix/onboarding/view/JunoOnboardingScreen.kt

```diff
@@ -115,10 +113,6 @@ fun JunoOnboardingScreen(
     LaunchedEffect(isWidgetPinnedState) {
         if (isWidgetPinnedState) {
             scrollToNextPageOrDismiss()
-            telemetryRecorder.onAddSearchWidgetClick(
```

Comment:
Is there gonna be a different telemetry to when the widget is pinned?

Reply:
Not sure about this, I will check in with @dnarcese.